cmd/package-builder/package-builder.go

```diff
@@ -54,6 +54,11 @@ func runMake(args []string) error {
 			false,
 			"enable debug logging",
 		)
+		flContainerTool = flagset.String(
+			"container_tool",
+			"docker",
```

Comment:
Just a quick note for the team that if we merge this, we'll need to update our internal tooling to provide this default value since we don't call `package-builder` on the command line.